src/array/fixed_size_binary/mod.rs

```diff
@@ -87,6 +87,12 @@ impl FixedSizeBinaryArray {
 
 // must use
 impl FixedSizeBinaryArray {
+    /// Retrieve the size of the binary elements
+    /// in this ['FixedSizeBinaryArray']
+    pub fn fixed_size(&self) -> usize {
```

Comment:
```suggestion
    pub fn size(&self) -> usize {
```

(to be aligned with FixedSizedListArray)